tools/ci/run_tc.py

```diff
@@ -160,7 +160,7 @@ def install_chrome(channel):
 
 def start_xvfb():
     start(["sudo", "Xvfb", os.environ["DISPLAY"], "-screen", "0",
-           "%sx%sx%s" % (os.environ["SCREEN_WIDTH"],
+           "{}x{}x{}".format(os.environ["SCREEN_WIDTH"],
                          os.environ["SCREEN_HEIGHT"],
                          os.environ["SCREEN_DEPTH"])])
```

Comment:
I'm not sure pyupgrade wants to use `.format()` here. f-strings would work:

```suggestion
           f"{os.environ["SCREEN_WIDTH"]}x{os.environ["SCREEN_HEIGHT"]}x{os.environ["SCREEN_DEPTH"]}"])
```

If the line becomes too long, then extract `width`, `height` and `depth` variables instead.

Reply:
Yes, see my other comment about my hypothesis why this is happening.

It seems like this script has multiple cases it will not handle, and there are definitely string that are missed or replaced like this. I've reverted this change and kept only the changes that correct to f-strings. Maybe in the future it is worth combing the repo to change these or creating a script that handles this as expected.